src/lzqgame.ts

```diff
@@ -258,26 +258,28 @@ async function playerInitialBoard(
             return;
         }
 
-        myGame.playerToSocketIdMap.forEach((socketId: string, instPlayerName: string) => {
-            console.info(
-                `Sending board to ${instPlayerName} on socket: ${socketId}`,
-            );
+        myGame.playerToSocketIdMap.forEach(
+            (socketId: string, instPlayerName: string) => {
+                console.info(
+                    `Sending board to ${instPlayerName} on socket: ${socketId}`,
+                );
 
-            if (!myGame) {
-                console.error('Game not found.');
-                this.emit('error', [`$Game not found: ${room}`]);
-                return;
-            }
+                if (!myGame) {
+                    console.error('Game not found.');
+                    this.emit('error', [`$Game not found: ${room}`]);
+                    return;
+                }
 
-            const playerIndex = myGame.players.indexOf(instPlayerName);
-            console.info(`playerIndex: ${playerIndex}`);
+                const playerIndex = myGame.players.indexOf(instPlayerName);
+                console.info(`playerIndex: ${playerIndex}`);
```

Comment:
some weird formatting happening here?

Reply:
Look at the file: https://github.com/chinese-board-games/luzhanqi-backend/blob/4ac44a11d0e151e30bd8bb772e5d700ddc11651a/src/lzqgame.ts